app/src/test/java/com/marvelapp/presentation/viewmodel/SplashViewModelTest.kt

```diff
@@ -0,0 +1,44 @@
+package com.marvelapp.presentation.viewmodel
+
+import androidx.arch.core.executor.testing.InstantTaskExecutorRule
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.test.*
+import org.junit.*
+import org.junit.rules.TestRule
+import org.junit.runner.RunWith
+import org.mockito.junit.MockitoJUnitRunner
+
+@RunWith(MockitoJUnitRunner::class)
+@OptIn(ExperimentalCoroutinesApi::class)
+class SplashViewModelTest {
+
+    private val splashViewModel = SplashViewModel()
+
+    private val testDispatcher = StandardTestDispatcher()
+
+    @get:Rule
+    val taskExecutorRule = InstantTaskExecutorRule()
+
+    @Before
+    fun init() {
+        Dispatchers.setMain(testDispatcher)
+    }
+
+    @After
+    fun after() {
+        Dispatchers.resetMain()
```

Comment:
here, you missed the line 
`testDispatcher.cleanupTestCoroutines()`

Reply:
Solved